config/default.js

```diff
@@ -22,46 +22,46 @@ module.exports = {
   //
   // Server
   //
-  server: {
-    port: 3000,
-    router: {
-      stripTrailingSlash: true
+  server : {
+    port   : 3000,
```

Comment:
The Airbnb's style guide had `alignment` on `:` but I didnt like it because without the space before it looked not so good  😛, but you can changed it to your pref, is this line https://github.com/AvraamMavridis/hapijs-seed-mvc/blob/master/.eslintrc#L33
and this rule
http://eslint.org/docs/rules/key-spacing

And if you run `eslint --fix .`  eslint will automatically fix it in every file. :)